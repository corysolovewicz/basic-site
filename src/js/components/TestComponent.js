import React from 'react'

const TestComponent = () => (
  <div>
    <ul>
      <li>This</li>
      <li>Is</li>
      <li>Just</li>
      <li>A</li>
      <li>Test</li>
    </ul>
  </div>
)

export default TestComponent