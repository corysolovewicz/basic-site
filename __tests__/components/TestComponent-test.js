import React from 'react';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router';

import TestComponent from '../../src/js/components/TestComponent';

// needed because this:
// https://github.com/facebook/jest/issues/1353
jest.mock('react-dom');

test('App renders', () => {
  const component = renderer.create(
    <MemoryRouter>
      <TestComponent />
    </MemoryRouter>
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
