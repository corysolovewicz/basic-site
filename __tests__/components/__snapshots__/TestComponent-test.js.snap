// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders 1`] = `
<div
  className="grommet grommetux-app"
  lang="en-US"
>
  <div
    className="grommetux-box grommetux-box--direction-column grommetux-box--responsive grommetux-box--pad-none grommetux-box--flex-off"
    id={undefined}
    onClick={undefined}
    role={undefined}
    style={Object {}}
    tabIndex={undefined}
  >
    <header
      className="grommetux-box grommetux-box--direction-row grommetux-box--align-center grommetux-box--pad-horizontal-none grommetux-box--pad-vertical-none grommetux-box--pad-between-small grommetux-box--pad-between-thirds grommetux-background-color-index-light-2 grommetux-background-color-index--light grommetux-header grommetux-header--small"
      id={undefined}
      onClick={undefined}
      role={undefined}
      style={Object {}}
      tabIndex={undefined}
    >
      <h1
        className="grommetux-heading grommetux-heading--align-center"
      />
      <div
        className="grommetux-box grommetux-box--direction-row grommetux-box--justify-end grommetux-box--flex grommetux-box--pad-none"
        id={undefined}
        onClick={undefined}
        role={undefined}
        style={Object {}}
        tabIndex={undefined}
      />
      <div
        className="grommetux-box grommetux-box--direction-column grommetux-box--responsive grommetux-box--pad-none grommetux-menu grommetux-menu--column grommetux-menu--controlled grommetux-menu__control"
        id={undefined}
        onClick={undefined}
        role={undefined}
        style={Object {}}
        tabIndex={undefined}
      >
        <button
          aria-label="Open  Menu"
          className="grommetux-button grommetux-button--plain"
          disabled={false}
          href={undefined}
          onBlur={[Function]}
          onClick={[Function]}
          onFocus={[Function]}
          onMouseDown={[Function]}
          onMouseUp={[Function]}
          type="button"
        >
          <span
            className="grommetux-button__icon"
          >
            <svg
              aria-label="actions"
              className="grommetux-control-icon grommetux-control-icon-actions grommetux-control-icon--responsive"
              height="24px"
              role="img"
              version="1.1"
              viewBox="0 0 24 24"
              width="24px"
            >
              <path
                d="M12,17.5 C15.0375661,17.5 17.5,15.0375661 17.5,12 C17.5,8.96243388 15.0375661,6.5 12,6.5 C8.96243388,6.5 6.5,8.96243388 6.5,12 C6.5,15.0375661 8.96243388,17.5 12,17.5 Z M12,6.5 L12,1 M12,23 L12,17.5 M1,12 L6.5,12 M17.5,12 L23,12 M4.4375,4.4375 L8.5625,8.5625 M15.4375,15.4375 L19.5625,19.5625 M19.5625,4.4375 L15.4375,8.5625 M8.5625,15.4375 L4.4375,19.5625"
                fill="none"
                stroke="#000"
                strokeWidth="2"
              />
            </svg>
          </span>
        </button>
      </div>
    </header>
  </div>
  <div
    align={
      Object {
        "top": true,
      }
    }
    className="grommetux-hero grommetux-hero--small grommetux-background-color-index--dark"
  >
    <div
      className="grommetux-hero__background"
    >
      <img
        className="grommetux-image grommetux-image--medium grommetux-image--cover grommetux-image--full"
        src="/img/hexdump_genesis_block_cory_solovewicz.png"
      />
    </div>
    <div
      className="grommetux-hero__foreground"
    >
      <div
        className="grommetux-box grommetux-box--direction-row grommetux-box--justify-center grommetux-box--align-center grommetux-box--responsive grommetux-box--pad-none"
        id={undefined}
        onClick={undefined}
        role={undefined}
        style={Object {}}
        tabIndex={undefined}
      >
        <div
          className="grommetux-box grommetux-box--direction-column grommetux-box--align-end grommetux-box--responsive grommetux-box--basis-1-2 grommetux-box--pad-medium"
          id={undefined}
          onClick={undefined}
          role={undefined}
          style={Object {}}
          tabIndex={undefined}
        />
        <div
          className="grommetux-box grommetux-box--direction-column grommetux-box--align-start grommetux-box--responsive grommetux-box--basis-1-2 grommetux-box--pad-medium"
          id={undefined}
          onClick={undefined}
          role={undefined}
          style={Object {}}
          tabIndex={undefined}
        >
          <div
            className="grommetux-box grommetux-box--direction-column grommetux-box--responsive grommetux-box--pad-none grommetux-box--wrap grommetux-box--width-medium grommetux-box--size grommetux-card"
            id={undefined}
            onClick={undefined}
            role={undefined}
            style={Object {}}
            tabIndex={undefined}
          >
            <div
              className="grommetux-box grommetux-box--direction-column grommetux-box--responsive grommetux-box--pad-medium grommetux-card__content"
              id={undefined}
              onClick={undefined}
              role={undefined}
              style={Object {}}
              tabIndex={undefined}
            >
              <label
                className="grommetux-label grommetux-label--uppercase grommetux-label--margin-none grommetux-label--medium"
                htmlFor={undefined}
              >
                Cory
              </label>
              <h2
                className="grommetux-heading grommetux-heading--strong"
              >
                Solovewicz
              </h2>
              <p
                className="grommetux-paragraph grommetux-paragraph--large grommetux-paragraph--margin-medium"
              >
                Fullstack Developer, Crypto Geek, And Contributor to Open Source
              </p>
              <a
                aria-label={undefined}
                className="grommetux-anchor grommetux-anchor--animate-icon grommetux-anchor--icon grommetux-anchor--icon-label grommetux-anchor--primary"
                href="mailto:cory@solovewi.cz"
                onClick={[Function]}
              >
                <svg
                  aria-label="link next"
                  className="grommetux-control-icon grommetux-control-icon-link-next grommetux-control-icon--responsive"
                  height="24px"
                  role="img"
                  version="1.1"
                  viewBox="0 0 24 24"
                  width="24px"
                >
                  <path
                    d="M2,12 L22,12 M13,3 L22,12 L13,21"
                    fill="none"
                    stroke="#000"
                    strokeWidth="2"
                  />
                </svg>
                Contact Me
              </a>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div>
    <ul>
      <li>
        This
      </li>
      <li>
        Is
      </li>
      <li>
        Just
      </li>
      <li>
        A
      </li>
      <li>
        Test
      </li>
      <li>
        Weeee!
      </li>
    </ul>
  </div>
  <footer
    className="grommetux-box grommetux-box--direction-row grommetux-box--justify-between grommetux-box--align-center grommetux-box--pad-none grommetux-footer"
    id={undefined}
    onClick={undefined}
    role={undefined}
    style={Object {}}
    tabIndex={undefined}
  >
    <div
      className="grommetux-box grommetux-box--direction-row grommetux-box--align-center grommetux-box--responsive grommetux-box--pad-between-medium"
      id={undefined}
      onClick={undefined}
      role={undefined}
      style={Object {}}
      tabIndex={undefined}
    >
      <p
        className="grommetux-paragraph grommetux-paragraph--margin-none"
      >
        Copyright Â© 
        2018
         Cory Solovewicz
      </p>
    </div>
  </footer>
  <span
    style={
      Object {
        "display": "none",
      }
    }
  />
  <div
    aria-live="polite"
    className="grommetux-app__announcer"
  />
</div>
`;
